bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/capture/digitalinvoice/view/DigitalInvoiceOnboardingBarBottomAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package net.gini.android.bank.sdk.capture.digitalinvoice.view
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import net.gini.android.bank.sdk.databinding.GbsDigitalInvoiceNavigationBarBottomBinding
+import net.gini.android.capture.view.InjectedViewAdapter
+
+interface DigitalInvoiceOnboardingBarBottomAdapter: InjectedViewAdapter {
```

Comment:
[Request] Please rename it to "DigitalInvoiceOnboarding**Navigation**BarBottomAdapter".